airsonic-main/src/main/java/org/airsonic/player/service/search/IndexManager.java

```diff
@@ -306,8 +306,15 @@ private void stopIndexing(IndexType type, MediaLibraryStatistics statistics) {
             File indexDirectory = getIndexDirectory.apply(indexType);
             try {
                 if (indexDirectory.exists()) {
-                    SearcherManager manager = new SearcherManager(FSDirectory.open(indexDirectory.toPath()), null);
-                    searchers.put(indexType, manager);
+                    SearcherManager manager = null;
+                    try {
+                        manager = new SearcherManager(FSDirectory.open(indexDirectory.toPath()), null);
+                    } catch (IndexNotFoundException e) {
+                        LOG.debug("Index {} does not exist in {}, likely not yet created.", indexType.toString(), indexDirectory.getAbsolutePath());
```

Comment:
This exception is already caught here: https://github.com/airsonic/airsonic/pull/1459/files#diff-4bd938d8bb34da2b664e6656a8c56847R322

You could just deprecate the error there to a lower log level, since that's what you're doing here too.

Alternatively add another (higher) catch clause there with a more specific IOException. Multiple nested try-catches make code reading a bit harder.

Reply:
The link doesn't work for me... Can you check that?

Reply:
If the process exits promptly with a specific exception, it will return null.
```
} catch (IndexNotFoundException e) {
    LOG.debug(....);
    retrun null;
}
```

Reply:
Hmmm, not sure why the link doesn't work, but it's L315 of the original (L322 of the modified) where the exception is caught.

You could just modify log level to debug (or warn) instead of error.

My other suggestion was demonstrated by @tesshucom above

Reply:
Ah, thank you both, good suggestions. You're right, it's way better without the nesting, I updated the PR!